adam_core/orbits/orbits.py

```diff
@@ -111,18 +111,16 @@ def to_df(
         coordinate_type : {"cartesian", "spherical", "keplerian", "cometary"}
             Desired output representation of the orbits.
         sigmas : bool, optional
-            Include 1-sigma uncertainty columns. If None, will determine if
-            any uncertainties are present and add them if so.
+            Include 1-sigma uncertainty columns.
         covariances : bool, optional
-            Include lower triangular covariance matrix columns. If None, will
-            determine if any uncertainties are present and add them if so.
+            Include lower triangular covariance matrix columns.
 
         Returns
         -------
         df : `~pandas.DataFrame`
             Pandas DataFrame containing orbits.
         """
-        df = CoordinateMembers.to_df(
+        df = CoordinateMembers._to_df(
```

Comment:
Why explicitly call the parent class here?

Reply:
At some point, there was some ambiguity as two parent classes had `.to_df` implemented but that has been fixed.